src/context/monika-flags.ts

```diff
@@ -60,6 +60,7 @@ export type MonikaFlags = {
   text?: string
   verbose: boolean
   version: void
+  'experimental-probe-splitting'?: boolean
```

Comment:
IMO, it would be better to add `symon` prefix. For instance `experimental-symon-probe-splitting` or `symon-experimental-probe-splitting`. So that, all the Symon flag can grouped by the prefix.

Reply:
Updated to `symon-experimental-probe-splitting`.